lib/memcached.js

```diff
@@ -913,7 +913,7 @@ Client.config = {
     value = Utils.escapeValue(value);
 
     length = Buffer.byteLength(value);
-    if (length > this.maxValue) {
+    if (length + fullkey.length + 71 > this.maxValue) {
```

Comment:
A comment explaining the logic behind this (and the magic constant that is 71) would be appreciated :)